src/main/java/seedu/address/logic/command/copy/MoveCommand.java

```diff
@@ -61,7 +61,11 @@ public CommandResult execute(Model model) throws CommandException {
             model.copyActivity(activityToMove, travelPlanIndex);
             model.deleteActivity(activityToMove);
 
-            return new CommandResult(String.format(MESSAGE_MOVE_ACTIVITY_SUCCESS, activityToMove, travelPlan));
+            assert travelPlan.getActivityList().hasActivity(activityToMove) : "Activity was not moved";
+            assert !model.getWishlist().hasActivity(activityToMove) : "Activity was not deleted after moving";
```

Comment:
can replace this with:

`assert !model.hasActivity(activityToMove) : "Activity was not deleted after moving";`

This is an already existing method in model.